quest/score-quest.js

```diff
@@ -0,0 +1,9 @@
+  
+function scoreQuest(choice, questId, user) {
+
+    user.madness += choice.madness;
+    user.bank += choice.bank;
```

Comment:
I don't see that any of your choices have a `bank` key so this is why `undefined` keeps being added to your bank field.